packages/react/src/components/Button/StyledButton.tsx

```diff
@@ -41,12 +42,7 @@ export const StyledButton = styled(Clickable)<StyledButtonProps>`
 
   &:not(:disabled):not([aria-disabled]),
   &[aria-disabled='false'] {
-    &:active,
```

Comment:
Is the focus ring style not needed for `active`?

Reply:
Yes, we have decided to display the focus ring only in the case of 'focus-visible'.